formidable/views.py

```diff
@@ -246,12 +246,14 @@ def get(self, request, **kwargs):
             raise exceptions.NotFound()
 
         form_class = self.get_form_class(formidable)
-        form = self.get_form(form_class)
+        form = self.get_form(form_class, request)
         if form.is_valid():
             return self.form_valid(form)
         else:
             return self.form_invalid(form)
 
+    post = get
+
```

Comment:
I would like to inverse the methods (as we will remove the GET method).
```
def post(self, request, **kwargs): 
     # all the code goes here

get = post

```

Reply:
Done, also added a deprecation warning as @brunobord suggested!